backbone.js

```diff
@@ -1455,6 +1455,16 @@
 
   // Set up all inheritable **Backbone.Router** properties and methods.
   _.extend(Router.prototype, Events, {
+    // If set to true, route handlers will be called with routeData object
+    //
+    //     this.route('search/:query/p:num', 'search', function(routeData) {
```

Comment:
I'd be ok with `(params, routeData)`. The most important piece of information here is the `params` object, hiding it inside `routeData` just adds friction.